GitUI/Translation/English.xlf

```diff
@@ -1358,6 +1358,10 @@ The primary difftool can still be selected by clicking the main menu entry.</sou
         <source>No changes</source>
         <target />
       </trans-unit>
+      <trans-unit id="_openInVisualStudioMenuItem.Text">
```

Comment:
Unused translation item - should not be needed

Reply:
This is also true of the sibling `_openSubmoduleMenuItem` translation. I've renamed to prefix with `_NO_TRANSLATE_`.